SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -219,6 +222,29 @@ void InterestFactory::AddServerSelfInterest(Interest& OutInterest, const Worker_
 	AddComponentQueryPairToInterestComponent(OutInterest, SpatialConstants::ENTITY_ACL_COMPONENT_ID, LoadBalanceQuery);
 }
 
+bool InterestFactory::AddServerOwnerInterest(Interest& OutInterest, const AActor* InActor) const
```

Comment:
I presume the name is meant to mean - add owner interest on server?
I got slightly confused between this and - add interest appropriate for a server that owns the object.